src/index.js

```diff
@@ -553,53 +582,23 @@ function create(options) {
     return Promise.reject(new DropinError('options.authorization is required.'));
   }
 
-  return client.create({
-    authorization: options.authorization
-  }).catch(function (err) {
-    return Promise.reject(new DropinError({
-      message: 'There was an error creating Drop-in.',
-      braintreeWebError: err
-    }));
-  }).then(function (clientInstance) {
-    clientInstance = setAnalyticsIntegration(clientInstance);
-
-    if (clientInstance.getConfiguration().authorizationType === 'TOKENIZATION_KEY') {
-      analytics.sendEvent(clientInstance, 'started.tokenization-key');
-    } else {
-      analytics.sendEvent(clientInstance, 'started.client-token');
-    }
+  analytics.setupAnalytics(options.authorization);
 
-    return new Promise(function (resolve, reject) {
-      new Dropin({
-        merchantConfiguration: options,
-        client: clientInstance
-      })._initialize(function (err, instance) {
-        if (err) {
-          reject(err);
+  return new Promise(function (resolve, reject) {
+    new Dropin({
+      merchantConfiguration: options
+    })._initialize(function (err, instance) {
+      if (err) {
+        reject(err);
 
-          return;
-        }
+        return;
+      }
 
-        resolve(instance);
-      });
+      resolve(instance);
     });
   });
 }
 
-function setAnalyticsIntegration(clientInstance) {
-  var configuration = clientInstance.getConfiguration();
-
-  configuration.analyticsMetadata.integration = constants.INTEGRATION;
-  configuration.analyticsMetadata.integrationType = constants.INTEGRATION;
-  configuration.analyticsMetadata.dropinVersion = VERSION;
-
-  clientInstance.getConfiguration = function () {
-    return configuration;
-  };
-
-  return clientInstance;
-}
```

Comment:
Noticed this was removed entirely. Is analytics integration set via deferred client now?